core/templates/pages/about-page/about-page.component.html

```diff
@@ -1,261 +1,1060 @@
-<div class="oppia-about-page-container">
-  <div class="oppia-static-header">
-    <h1 [innerHTML]="'I18N_ABOUT_PAGE_HEADING' | translate"></h1>
+<div class="about-page">
+  <div class="oppia-about-page-section-one">
```

Comment:
@ReshuKumari Are the 1st, 3rd, and 4th sections of the page have the same structure? If yes can we create an about-page section- component and reuse it here? (This will help us reduce the HTML, CSS and different selectors)

Reply:
@DubeySandeep section 1st, 3rd and 4th exactly don't have same structure. As for 1st section the image covers the whole right-most part  whereas this is not the case with 3rd and 4th sections. And also 3rd section has no button whereas other section have buttons. And I don't think it will greatly impact the number of selectors, we will have to anyway apply the styling in anycase which the page needs. And also other pages follow the similar pattern only as done now hence it creates consistency in the codebase and easy to understand for the maintainer. So I don't think it would be a really great idea to do this.. as we can achieve a single thing by many ways and this way too shall be fine.

Reply:
Can we have this tag:
`<oppia-about-section i18n-heading='...' i18n-paragraph='...' image-filename='...' expanded-image button-required button-i18n-text='...'></oppia-about-section>`

I think this will help us in multiple ways:
1. Help us avoid different selectors like `.oppia-about-page-section-two`, `.oppia-about-page-section-two-top` etc. which are "section specific" selectors.
2. Have a better and clear structure for the about page.
3. Size of the HTML pages will be reduced (significantly) and will have a reusable structure.

@ReshuKumari Do you agree with the above? let me know what do you think? :)